Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/radar.py

```diff
@@ -159,6 +159,9 @@ def __init__(self, client, issue):
         else:
             self.component = None
 
+        if 'sourceChanges' in (additional_fields or []):
```

Comment:
Nit: although a test mock test case, would be nice to set `additional_fields = additional_field or []` right away in case it's used later on.